oracle/check.py

```diff
@@ -0,0 +1,127 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+
+# 3rd party
+try:
+    import cx_Oracle
+except Exception as e:
+    cx_Oracle = None
+
+# project
+from checks import AgentCheck
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'oracle'
+
+
+class Oracle(AgentCheck):
+
+    SERVICE_CHECK_NAME = 'oracle.can_connect'
+    SYS_METRICS = {
+        'Buffer Cache Hit Ratio':           'oracle.buffer_cachehit_ratio',
+        'Cursor Cache Hit Ratio':           'oracle.cursor_cachehit_ratio',
+        'Library Cache Hit Ratio':          'oracle.library_cachehit_ratio',
+        'Shared Pool Free %':               'oracle.shared_pool_free',
+        'Physical Reads Per Sec':           'oracle.physical_reads',
+        'Physical Writes Per Sec':          'oracle.physical_writes',
+        'Enqueue Timeouts Per Sec':         'oracle.enqueue_timeouts',
+        'GC CR Block Received Per Second':  'oracle.gc_cr_receive_time',
+        'Global Cache Blocks Corrupted':    'oracle.cache_blocks_corrupt',
+        'Global Cache Blocks Lost':         'oracle.cache_blocks_lost',
+        'Logons Per Sec':                   'oracle.logons',
+        'Average Active Sessions':          'oracle.active_sessions',
+        'Long Table Scans Per Sec':         'oracle.long_table_scans',
+        'SQL Service Response Time':        'oracle.service_response_time',
+        'User Rollbacks Per Sec':           'oracle.user_rollbacks',
+        'Total Sorts Per User Call':        'oracle.sorts_per_user_call',
+        'Rows Per Sort':                    'oracle.rows_per_sort',
+        'Disk Sort Per Sec':                'oracle.disk_sorts',
+        'Memory Sorts Ratio':               'oracle.memory_sorts_ratio',
+        'Database Wait Time Ratio':         'oracle.database_wait_time_ratio',
+        'Enqueue Timeouts Per Sec':         'oracle.enqueue_timeouts',
+        'Session Limit %':                  'oracle.session_limit_usage',
+        'Session Count':                    'oracle.session_count',
+        'Temp Space Used':                  'oracle.temp_space_used',
+    }
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
```

Comment:
Nit: if you're just calling super you could presumably remove this

Reply:
Indeed!